packages/mechanic-engine-react/index.js

```diff
@@ -74,9 +70,8 @@ export default _default;
     return;
   }
 
-  reactHotLoader.register(root, "root", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-react-engine/src/index.js");
-  reactHotLoader.register(run, "run", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-react-engine/src/index.js");
-  reactHotLoader.register(_default, "default", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-react-engine/src/index.js");
+  reactHotLoader.register(root, "root", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-engine-react/src/index.js");
+  reactHotLoader.register(run, "run", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-engine-react/src/index.js");
```

Comment:
Are these lines to make hot module loader work on development? We should probably figure out a way to do this without hardcoding the user path.

Reply:
Nice observation. I didn't realize it was there since its the babel compilation output. That code is produced by the `react-hot-loader/babel` plugin, which I left in the `.babelrc` file as template from the template package. I read a little bit, and realized it not used by that package and I don't think it's needed. I removed it in the latest commit.